### DISCUSSION TITLE
Keine Verbindung möglich seit letztem Update

### DISCUSSION THREAD

**Discussion by LJ9210** [tbnobody/OpenDTU#232] [Q&A]
Habe heute das aktuelle Update aufgespielt jedoch seitdem auch keine Möglichkeit mehr auf OpenDTU zuzugreifen. 

Sobald ich die IP im Browser eingebe wird keine Verbindung hergestellt,auch im Router wird nichts bezüglich DTU aufgelistet. 

Hat jemand eine Idee wie das Problem behoben werden kann ?

**Comment by FrodoVDR** [comment]
Ich würde es mal mit einem USB Kabel an einem PC versuchen um die Console zu sehen. Oder falls du MQTT aktiviert hast schauen z.B. mit dem MQTT-Explorer ob dort etwas ankommt, dann solltest du nochmals schauen ob dein esp nicht eine andere IP-Adresse hat.
